frontend/src/locales/en.json

```diff
@@ -88,10 +89,13 @@
     "addCalendar": "Add {provider} calendar",
     "addDay": "Add day",
     "addTime": "Add time",
+    "admin-invite-codes-panel": "Invites",
+    "admin-subscriber-panel": "Subscribers",
```

Comment:
A bit odd to switch from camelCase to dash-case, but I guess that's because of the route names, right?

Reply:
Yep! I should just group those. 🤔

Reply:
Good idea, maybe just a section directly for route names...?

Reply:
Yep, I'll make a ticket for it.